src/Files.Shared/FileTagsDb.cs

```diff
@@ -167,8 +170,8 @@ public class TaggedFile
             [BsonId]
             public int Id { get; set; }
             public ulong? Frn { get; set; }
-            public string FilePath { get; set; }
-            public string Tag { get; set; }
+            public string FilePath { get; set; } = null!;
+            public string Tag { get; set; } = null!;
```

Comment:
If those properties can be assigned null, should they be marked as nullable?

Reply:
Better string.empty?

Reply:
Yeah maybe that would be better.